README.md

```diff
@@ -252,7 +252,10 @@ elastically:
                 transport:           '@JoliCode\Elastically\Transport\HttpClientTransport'
 
             # Path to the mapping directory (in YAML)
-            mapping_directory: '%kernel.project_dir%/config/elasticsearch'
+            mapping_directory:       '%kernel.project_dir%/config/elasticsearch'
+            
```

Comment:
Please please please, configure your IDE to automatically remove trailing white space